ow_lander/scripts/arm_action_servers.py

```diff
@@ -207,6 +211,7 @@ def on_stop_action(self, goal):
         trajectory_async_executer.stop()
 
         rospy.loginfo('%s: Succeeded' % self._action_name)
+        rospy.loginfo('Current Arm Action was stopped.')
```

Comment:
Since there's also an action-specific "was stopped" message, this one seems redundant.

Reply:
action specific stop could only be issued during planning phase... During execution all the plans use the same trajectory_async_executer. I added this to clarify that the current action was stopped and Stop Action succeeded.

Reply:
Got it.  It may be useful to add "during execution" to this message, and "during planning" in the specific messages.

Reply:
> Got it. It may be useful to add "during execution" to this message, and "during planning" in the specific messages.

The problem is during Stow and Unstow there is no separate planning and execution phase.